### ISSUE TITLE
Extension does not work when ordering view

### ISSUE THREAD

**Issue by ViniciusRed** [AykutSarac/jsoncrack-vscode#2]
* When asking to view the json the extension does not read the file when updating the file the view starts working
![image](https://user-images.githubusercontent.com/81523685/191141197-cc8b7b35-41c8-4bfa-9878-577f92278956.png)
* Update json
![image](https://user-images.githubusercontent.com/81523685/191141375-f6fbc5b0-a1f3-45de-a439-446904ce3c86.png)
* I sent my friend to do a test, he didn't have this problem, he uses Windows 10 22H2

![image](https://user-images.githubusercontent.com/81523685/191142080-c4a4988a-a718-4939-b695-2abed3d91017.png)
* I use Windows 8.1 Version 6.3 (Build 9600)

**Comment by AykutSarac** [comment]
I've pushed a fix, can you try it and let me know if it works for you?

**Comment by ViniciusRed** [comment]
Still having the same problem and now a black bar appears

**Comment by ViniciusRed** [comment]
![image](https://user-images.githubusercontent.com/81523685/192327700-f4e551b4-5584-4b75-aac8-cc20a5056e19.png)

**Comment by ViniciusRed** [comment]
I would like to know if jsoncrack-vscode has support for linux?
![image](https://user-images.githubusercontent.com/81523685/196585532-68a673d5-f7f5-44e8-93cd-98f729c37ab0.png)

**Comment by tehe** [comment]
FWIW, I've installed the extension to VS Code on Windows, yet the IDE is connected to a remote machine running Linux (Debian Bullseye) - that's where I've got my code and the tools etc. The extension did complain during installation (local and remote) and doesn't show any error or warning when trying to visualize a JSON file, but it also doesn't show any graph.

**Comment by amsteel** [comment]
Tried on both Windows and Ubuntu 22.04, not working. 
Ubuntu 22.04:
Version: 1.73.0
Commit: 8fa188b2b301d36553cbc9ce1b0a146ccb93351f
Date: 2022-11-01T15:44:09.336Z
Electron: 19.0.17
Chromium: 102.0.5005.167
Node.js: 16.14.2
V8: 10.2.154.15-electron.0
OS: Linux x64 5.15.0-30-generic
Sandboxed: No

Windows 10:
Version: 1.73.0 (user setup)
Commit: 8fa188b2b301d36553cbc9ce1b0a146ccb93351f
Date: 2022-11-01T15:34:06.111Z
Electron: 19.0.17
Chromium: 102.0.5005.167
Node.js: 16.14.2
V8: 10.2.154.15-electron.0
OS: Windows_NT x64 10.0.19044
Sandboxed: No

**Comment by fuxey** [comment]
same issue here on ubuntu - using vs code 1.73

**Comment by ViniciusRed** [comment]
Extension working normally on my ArchLinux, I changed my main system, I don't use Windows anymore.

Before it didn't work on my ArchLinux but I think after installing nodejs package and java 8 or java 17 or java 19 it made it work without problems.

I don't know if the issue I was having on Windows 8.1 was resolved, of the extension.

![image](https://user-images.githubusercontent.com/81523685/203897945-65d28ea4-3ba6-4873-99c7-621e86b29d4b.png)
